ansible_runner/runner_config.py

```diff
@@ -369,6 +386,15 @@ def build_process_isolation_temp_dir(self):
         os.chmod(path, stat.S_IRUSR | stat.S_IWUSR | stat.S_IXUSR)
         return path
 
+    def wrap_args_with_cgexec(self, args):
+        '''
+        Wrap existing command line with cgexec in order to profile resource usage
+        '''
+        # TODO: Should only wrap if `self.execution_mode == ExecutionMode.ANSIBLE_PLAYBOOK` (see example below)
```

Comment:
Does the callback plugin not work for adhoc commands?  Maybe we just don't care about those, considering their not likely critical from a performance tracking standpoint.

Reply:
@ryanpetrello - the plugin doesn't work with ad hoc commands. It only collects metrics when `v2_playbook_on_task_start` and `v2_playbook_on_stats` are called.

https://github.com/ansible/ansible/blob/devel/lib/ansible/plugins/callback/cgroup_perf_recap.py#L402-L405

Reply:
👍